src/graph.rs

```diff
@@ -3399,7 +3520,8 @@ fn fill_module_dependencies(
       // TODO(nayeemrmn): Import attributes should be visited and checked for
       // every import, not one per specifier.
```

Comment:
Unrelated to this PR, but do we track this anywhere? Seems like something that might pop up when using text or bytes imports?